chia/full_node/full_node.py

```diff
@@ -1197,17 +1262,32 @@ async def validate_block_batches(
                 # height, in that case.
                 self.blockchain.clean_block_record(end_height - self.constants.BLOCKS_CACHE_SIZE)
 
-        batch_queue_input: asyncio.Queue[Optional[tuple[WSChiaConnection, list[FullBlock]]]] = asyncio.Queue(
-            maxsize=buffer_size
-        )
-        fetch_task = asyncio.Task(fetch_block_batches(batch_queue_input))
-        validate_task = asyncio.Task(validate_block_batches(batch_queue_input))
+        block_queue: asyncio.Queue[Optional[tuple[WSChiaConnection, list[FullBlock]]]] = asyncio.Queue(maxsize=10)
+        validation_queue: asyncio.Queue[
+            Optional[tuple[WSChiaConnection, ValidationState, list[Awaitable[PreValidationResult]], list[FullBlock]]]
+        ] = asyncio.Queue(maxsize=10)
```

Comment:
Do we have the rationale behind picking 10 as max size documented somewhere for these queues? otherwise I wonder if it's useful to give the magic number a name and a comment.

Reply:
Presumably, someone thought it was reasonable to keep, at most, 320 blocks queued up to be validated. I doubt it has been tested and tweaked. At this point, there are so many other problems causing sync to be slow that I would think this is pretty low down on the list.